api/src/routes/HttpRoutes.js

```diff
@@ -354,6 +365,10 @@ restify.serve(router, Statement, {
       .catch(err => next(err));
   },
 });
+restify.serve(router, AggregationProcessor, {
+  preUpdate: (req, res) => res.sendStatus(405),
```

Comment:
It's actually not the suggestion for this PR but for all places in code. Maybe it's better to create some constant with response codes and use it? Just like so:

```
export const HTTP_RESPONSE_CODES = {
  METHOD_NOT_ALLOWED: 405,
};
```
or
```
export const METHOD_NOT_ALLOWED = 405;
```
and then `res.sendStatus(HTTP_RESPONSE_CODES.METHOD_NOT_ALLOWED)` or `res.sendStatus(METHOD_NOT_ALLOWED)`

@ryansmith94 , what do you say?

Reply:
Yeah I agree, in the past we've used https://www.npmjs.com/package/http-status-codes